app/src/main/java/com/monstar_lab/sample/android/kotlin/MainActivity.kt

```diff
@@ -36,4 +37,7 @@ class MainActivity : AppCompatActivity() {
             else -> super.onOptionsItemSelected(item)
         }
     }
+
+    public fun aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/sample-android-kotlin/blob/0af9a7a0ee9f47008d106ba4a87d4471113d4c75/app/src/main/java/com/monstar_lab/sample/android/kotlin/MainActivity.kt#L41"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>